### ISSUE TITLE
Self Publishing with LeanPub

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/practical-full-stack-testing#24]
Should you design your own cover: https://medium.com/@samlaing/self-publishing-43d89e6ea260

**Comment by nelsonic** [comment]
Google Image search for "***practical***": https://www.google.co.uk/search?tbm=isch&q=practical

**Comment by iteles** [comment]
As repeating customers we can get a free _'Power Pack Upgrade'_ (basically our listing in bold and highlighted. We just need to launch it through dwyl.

**Comment by nelsonic** [comment]
That's the plan... :wink:

**Comment by nelsonic** [comment]
Created a specific issue for the 99Designs Cover Contest: https://github.com/nelsonic/practical-full-stack-testing/issues/25

**Comment by iteles** [comment]
Always! When do you want the cover to be ready for?

**Comment by nelsonic** [comment]
Yesterday! :wink:

**Comment by iteles** [comment]
:stuck_out_tongue: Let's talk about it over soup and I'll make it my task for tomorrow morning :kissing_heart:

**Comment by nelsonic** [comment]
Its *so hard* to prioritise things...! :persevere:
We need to get back on https://github.com/dwyl/time ***ASAP*** so we can prep for https://github.com/dwyl/summer-2015 ...
I've dusted this off so that I can prep for Tuesday's https://github.com/ladieswhocode/london meetup...

**Comment by iteles** [comment]
I reckon it makes sense for you to concentrate on this solidly for the next 3 days as it will allow you to make good progress on it as well as being prepped for [Tuesday's meetup](http://www.meetup.com/Ladies-Who-Code-UK/events/222452944/). So for the next 3 days we keep the [Summer stuff](https://github.com/dwyl/summer-2015) to conversations and planning that I can keep going with and then dive back into the code after Tuesday :+1:

**Comment by nelsonic** [comment]
Right... but we *also* need to Pitch the Summer to the FAC5 on Monday! :open_mouth:
So people can start planning if they want to join us in July or August... so that has to be a priority too...

**Comment by iteles** [comment]
I've literally just seen [issue #6](https://github.com/dwyl/summer-2015/issues/6). Wednesday might have been a better pick.
